.github/workflows/test.yaml

```diff
@@ -2,6 +2,9 @@ name: "Rspec for Rails Apps"
 on:
   workflow_dispatch:
     inputs:
+      confdir:
+        required: false
+        type: string
```

Comment:
can this have a default of the old value? that way we don't break existing builds and only override it when we need to.

Reply:
Thank you! Resolved!